src/packages/client/src/__tests__/runtime-tests/blog-env-postgresql/index.js

```diff
@@ -104,6 +104,9 @@ module.exports = async () => {
 
   await prisma.user.findMany()
   prisma.disconnect()
+
+  // flakyness :shrug:
+  await new Promise((r) => setTimeout(r, 200))
```

Comment:
Do you know waht's fleaky 👀

Reply:
So we're relying on everything being there in order, but depending on the order of the event loop / the stdio interface etc the order can be broken due to a race condition.